server/boot/index.js

```diff
@@ -6,26 +6,35 @@ var path = require( 'path' ),
 	config = require( 'config' ),
 	express = require( 'express' ),
 	morgan = require( 'morgan' ),
-	pages = require( 'pages' );
+	pages = require( 'pages' ),
+	ReactEngine = require( 'express-react-views' );
+
+var CALYPSO_ENV = process.env.CALYPSO_ENV || process.env.NODE_ENV || 'development';
```

Comment:
1. `config( 'env' )` should handle the fallbacks on your behalf
2. I think we can/should start using Node-supported variable declarations (i.e. `const`)